assets/js/app.js

```diff
@@ -0,0 +1,58 @@
+let numberRows = 0
+let stringOfData = []
+
+function getInformation() {
+    const name = document.getElementById(`name`).value
+    const price = document.getElementById(`price`).value
+    const chosenOption = document.getElementById(`option`).value
+    const options = [`Drinks`, `Meat`, `Dairy`, `Snacks`, `Basic`]
+    const option = options[chosenOption]
+    const url = document.getElementById(`url`).value
+    const numRows = numberRows++
+    if (name === `` || price === `` || url === ``) {
```

Comment:
several issues here:
1. as a user, I don't know how to fix the problem 
![Screen Shot 2024-11-25 at 11 24 02](https://github.com/user-attachments/assets/6015093b-a31a-44c0-8a7f-df056ad5c2d8)

2. you don't validate the `select` (i.e. product category)